dark.js

```diff
@@ -0,0 +1 @@
+Vars.state.rules.lighting
```

Comment:
```suggestion
Vars.state.rules.lighting=false
```

Reply:
see im a genius and never make mistakes abtuabsgakjbgl